example/example.json

```diff
@@ -182,6 +251,51 @@
           }
         }
       },
+      "example.FooMergePatch": {
+        "type": "object",
+        "properties": {
+          "count": {
+            "type": "integer",
+            "format": "int64"
+          }
+        }
+      },
+      "example.FooPatchOperation": {
+        "type": "object",
+        "properties": {
+          "op": {
+            "type": "string"
+          },
+          "path": {
+            "type": "string"
+          },
+          "value": {
+            "type": "string"
+          }
+        }
+      },
+      "example.FooPatchOperationSet": {
+        "type": "object",
+        "properties": {
+          "operations": {
+            "type": "array",
+            "items": {
+              "type": "object",
```

Comment:
Again, this should probably be a ref to `example.FooPatchOperation`

Reply:
Agreed, this should be pointing to that SchemaObject, I haven't been able to pinpoint the case yet either.

Reply:
Alright, I'll dig into it again today to see what I can do.

Reply:
Tested this on `master` and experienced the same issue, so not going to solve it in this PR. Created a ticket for it, https://app.clubhouse.io/launchdarkly/story/114017/goas-doesn-t-always-nest-schema